cmd/tdexd/main.go

```diff
@@ -70,8 +71,12 @@ func main() {
 		return
 	}
 
-	// TODO: setup a portable wallet
-	wallet, err := application.NewWallet(nil)
+	// TODO: move to config ? or constant ?
+	// who is running the ocean server ? the daemon ?
+	addrOceanServer := "localhost:50051"
```

Comment:
we will move this to config. At startup, if the daemon fails to connect to the ocean wallet, the process is stopped and the error logged to stdout.